### DISCUSSION TITLE
Is there issues using Snapshot with Trezor hard wallet ?

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#1189] [Q&A] [answered]
Vote for an answer below

**Comment by metrox-eth** [comment, marked as answer]
Atm you will only be able to vote using the Trezor (with Metamask) login option, if you are already logged in with Metamask, please logout and login again with the proper Trezor connector. You won't be able to join spaces as well as create a proposal exceeding 50 characters since it requiert EIP-712 message signing, Trezor plan an update that will solve the problem, it will be live in current of december 2021 for Trezor T and is planned in January 2022 for Trezor One, atm you can only vote using the Trezor (with Metamask) workaround login option.

**Reply by ys3627** [reply]
Wonderful project, this project very good and this projector has a lot of attractions, so hopefully the project will be better in the future and the cryptocurrency will be the best Number one
